### PR TITLE
Style front page

### PR DESCRIPTION
**Pull request by varffs**

### CODE REVIEW

File: functions.php [hunk 1]
```diff
@@ -123,6 +123,12 @@ function custom_login_logo() {
 add_action( 'login_head', 'custom_login_logo' );
 */
 
+// CUSTOM EXCERPT MORE TEXT
+function custom_excerpt_more( $more ) {
+	return ' <span class="font-small-caps">READ MORE</span>';
```
Comment (mixcas) [inline functions.php]: what about spanish?

Comment (varffs) [inline functions.php]: its says `read more` in the spanish designs!

Comment (mixcas) [inline functions.php]: i would think that's an error

Comment (varffs) [inline functions.php]: ok im happy to localize it

**Pull request merged**